meta-balena-common/recipes-bsp/u-boot/patches/env_resin.h

```diff
@@ -176,7 +176,7 @@
               "fi;\0" \
        \
        "set_os_cmdline=" \
-               "setenv os_cmdline ${base_os_cmdline} ${bootparam_flasher} ${extra_os_cmdline};\0" \
+               "setenv os_cmdline ${base_os_cmdline} ${bootparam_flasher} ${extra_os_cmdline} ${extra_os_firmware_class_path};\0" \
```

Comment:
> We use a separate u-boot env var for the search path
because in the future the extra_os_cmdline variable will be managed by
a separate configuration unit, based on a whitelist.

Is this why it is not stringified above with the other env vars on lines 33-50?

Reply:
hey @klutchell , only environment variables which have a build-time defined value are stringified - for example `resin_uboot_device_types` which is set to "mmc" and may be overridden at device repo level to add "usb" , "nvme", etc. In our case ${extra_os_firmware_class_path} is not defined at build-time, only at runtime and only if it is set in config.json.